### PR TITLE
Add C++ Solution + Editorial to Two Sets II (CSES)

### PR DESCRIPTION
**Pull request by ryanchou-dev**
ive been so inconsistent lately

---
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cses-1093.mdx
Comment (SansPapyrus683) [inline solutions/cses-1093.mdx:51]: just explain the array again here also remove the `, 0`

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/cses-1093.mdx
Comment (brad182) [inline solutions/cses-1093.mdx:13]:
```suggestion
Since the two sets have to have an equal sum, it's evident that both of them have to have half of  $1, 2, \ldots, N$. So, if $\frac{N(N+1)}{2}$, or the sum of the first $N$ positive integers isn't even, then a solution isn't possible. For each subset, we want its sum to be half of this sum, or $\frac{N(N+1)}{4}$
```

**Review by brad182 — APPROVED** [review]

File: solutions/cses-1093.mdx
Comment (ryanchou-dev) [reply solutions/cses-1093.mdx]: mb

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by SansPapyrus683** [comment]
i'll wait for the orca's approval then merge

**Review by TruaShamu — APPROVED** [review]
whale done :3

**Pull request merged**
